src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,25 @@
+fun main(args: Array<String>) {
+    scenario(12,8)
+}
+
+fun scenario(numberOfCars : Int, capacity : Int) {
+    /*
+    * Creates a given a number of cars, every other car wants to drive
+    * A road network is created with the given capacity
+    * The road network is analyzed and it is printed whether a car is delayed or not
+    */
+    val listOfCars = ListOfCars()
```

Comment:
Well, you need some _code_ to create the cars and put them into a list. I suggested to put that code not into the `Car` class but someplace else. The `scenario` method here is exactly the place for that.

I might have said, you need a class for that. If so, then I'm sorry for the confusion. I'm a Java developer and in Java everything is in a class. In Kotlin, on the other hand, you can have free floating functions like `scenario`.